addons/website/static/src/components/fullscreen_indication/fullscreen_indication.js

```diff
@@ -24,7 +24,7 @@ export class FullscreenIndication extends Component {
     }
 
     get fullScreenIndicationText() {
-        return markup(sprintf(escape(_t("Press %(key)s to exit full string")), "<span>esc</span>"));
+        return markup(sprintf(escape(_t("Press %(key)s to exit full screen")), {key: "<span>esc</span>"}));
```

Comment:
@odoo/rd-security Solved a merge conflict in this fw-port. Original override: https://github.com/odoo/odoo/pull/164893#issuecomment-2109599295